numba/cuda/tests/cudapy/test_dispatcher.py

```diff
@@ -92,20 +100,54 @@ def f(x, y):
 
         self.assertEqual(len(f.specializations), 0)
 
-        # 'A' order specialization
-        f_f32a_f32a = f.specialize(float32[:], float32[:])
+        # 'F' order specialization
+        f_arg = np.zeros((2, 2), order='F')
+        f_f32a_f32a = f.specialize(f_arg, f_arg)
```

Comment:
```suggestion
        f_f32f_f32f = f.specialize(f_arg, f_arg)
```
suggestion to match ordering `F`.